pkg/proxy/node.go

```diff
@@ -90,6 +92,79 @@ func (n *NodePodCIDRHandler) OnNodeDelete(node *v1.Node) {
 // OnNodeSynced is a handler for Node syncs.
 func (n *NodePodCIDRHandler) OnNodeSynced() {}
 
+// NodePodCIDRHandler handles the life cycle of kube-proxy based on the node PodCIDR assigned
+// Implements the config.NodeHandler interface
+// https://issues.k8s.io/111321
+type NodeIPsHandler struct {
```

Comment:
This doesn't seem to be used anywhere in the production code...

Reply:
OK - so it was supposed to be used but the name doesn't match - and it's what causing compile errors...

Reply:
yeah, heh, I have a problem that the hollow proxy fails to be initialized now